jenkins-pipelines/longevity-cdc-100gb-4h.jenkinsfile

```diff
@@ -6,6 +6,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 longevityPipeline(
     backend: 'aws',
     region: 'eu-west-1',
+    availability_zone: 'a,b,c',
```

Comment:
Why do we put it as a pipeline parameter and not in the test_yaml?
It's hard to track it like this.

Reply:
@roydahan 
because pipelines define availability_zone defaults - so it will override yaml.
Solutions I see:
1. add yet another param to test yaml which will be confusing (setting az in jenkins pipeline will differ from actual az's)
2. add availability_zone to test yaml and leave jenkins params as now (so it's clearer when reading test yaml) - my preference.

@roydahan please let me know if you have another idea or which you prefer

Reply:
Also the current disable of the new provision step depends on this pipeline parameter.

So we need some indication on the pipeline level as well

Reply:
I see.
I don't like option 1 either.
The problem with option 2 is that one may want to change that and he'll change that only in the yaml but it won't take effect since the pipeline overrides it, right?
(In that case, it doesn't really help and better to leave as is).